stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/attachment/AttachmentDialogStyle.kt

```diff
@@ -0,0 +1,35 @@
+package io.getstream.chat.android.ui.message.input.attachment
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.drawable.Drawable
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.common.extensions.internal.getDrawableCompat
+
+public data class AttachmentDialogStyle(
```

Comment:
What do you think about changing name to `AttachmentSelectionDialogStyle` just to match dialog's name?